cariban/templates/dataset/detail_html.mako

```diff
@@ -21,18 +26,19 @@ It is intended as a companion piece to my dissertation.
 
 For each language, there is a list of:
 
-<ul>
-<li> grammatical morphemes
-<li> verbal constructions, which contain person marking paradigms
-<li> t-adding verbs, an idiosyncratic lexical group of verbs
-<li> lexical morphemes, compiled for my <a href="https://gitlab.com/florianmatter/cariban_phylo/">phylogenetic classification</a>
-<li> example sentences
-<li> relevant literature
-</ul>
+<ol>
+<li> <a href="/morphemes">grammatical morphemes</a>
+<li> <a href="/constructions">verbal constructions</a>, which contain person marking paradigms
+<li> <a href="/morphemes#tadding">t-adding verbs</a>, a group of transitive verbs with an idiosyncratic 3P marker rc:t-
+<li> <a href="/morphemes#swadesh">lexical morphemes</a>, compiled for my <a href="https://gitlab.com/florianmatter/cariban_phylo/">phylogenetic classification</a>
+<li> <a href="/examples">example sentences</a>
+<li> relevant <a href="/sources">literature</a>
```

Comment:
While these links will work in our production setting, it would be more future-proof to construct them via
```python
req.route_url('morphemes')
req.route_url('morphemes', _anchor='tadding')
```
etc.

Reply:
Fixed.